microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -12,34 +12,37 @@ keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, I
 
 # Understand WebView2 SDK versions  
 
-WebView2 depends on Microsoft Edge to function.  Each WebView2 SDK requires that a minimum browser version is installed.  The minimum version is reflected in the package version of the SDK.  For example, if you use the `SDK package version 0.9.488`, then you must install Microsoft Edge with a build number of 488 or later.  The browser version is also specified in the WebView2 [Release Notes][Releasenotes].  For more information about the latest release of the Microsoft Edge browser, navigate to [Browser Channels][DeployedgeChannels].  
+To develop a WebView2 application, a minimum version of [WebView2 Runtime]() or a [non-stable Microsoft Edge channel]() must be installed.  The minimum version is reflected in the package version of the SDK.  For example, if you use the `SDK package version 0.9.488`, then you must install [WebView2 Runtime]() or a [non-stable Microsoft Edge channel]() with a build number of 488 or later.  The browser version is also specified in the WebView2 [Release Notes][Releasenotes].  For more information about the latest release of the Microsoft Edge browser, navigate to [Browser Channels][DeployedgeChannels].    
 
-> [!NOTE]
-> WebView2 is currently in preview.  While the WebView team strives to ensure backward compatibility between browser versions and SDKs, it is not guaranteed because newer versions of the browser may not support previous SDK versions.  If there are breaking changes between browser versions and SDKs, the changes are specified in the [Release Notes][Releasenotes].  
+New versions of the WebView2 SDK are shipped at the same cadence as the Microsoft Edge (Chromium) browser, which is approximately every 6 weeks. When a new SDK is shipped, it is compatible with the latest version of Microsoft Edge. Therefore, the WebView2 team recommends developing against the Microsoft Edge canary channel or the WebView2 Runtime. 
 
-In the future, the WebView team plans to change the distribution model for WebView2 apps.  For more information, navigate to [Evergreen distribution mode][DistributionEvergreenMode].  
+> [!IMPORTANT]
+> When developing WebView2 applications, regularly test your application against the latest versions of the WebView2 SDK. Because the web platform is constantly evolving, regular testing is the best way to ensure your application is performing as intended.
 
 ## Release and prerelease package  
 
-In preview, the release package contains the following.  
+The WebView2 Nuget package contains both a release and pre-release package.
 
-*   [Win32 C/C++ APIs][ReferenceWin32]: APIs in the SDK that are expected to remain the same at GA.  
+The release package contains the following.  
```

Comment:
Let's mention that starting with <whatever sdk version we GA with>, the Release SDKs are forward-compatible.